includes/definitions.inc.php

```diff
@@ -888,8 +888,8 @@
 $config['os'][$os]['over'][2]['text']  = 'Memory Usage';
 
 $os = 'fabos';
-$config['os'][$os]['text']             = 'Brocade FabricOS';
-$config['os'][$os]['type']             = 'network';
+$config['os'][$os]['text']             = 'FabricOS';
+$config['os'][$os]['type']             = 'storage';
```

Comment:
Anyone else got a comment on that - seems like it belongs in networking to me still as it's a switch.

@librenms/reviewers